api/python/cellxgene_census/src/cellxgene_census/_release_directory.py

```diff
@@ -133,3 +149,18 @@ def get_census_version_directory() -> Dict[CensusVersionName, CensusVersionDescr
 
     # Cast is safe, as we have removed all aliases
     return cast(Dict[CensusVersionName, CensusVersionDescription], directory)
+
+
+def get_census_mirrors() -> CensusMirrors:
+    response = requests.get(CELL_CENSUS_MIRRORS_DIRECTORY_URL)
+    response.raise_for_status()
+    return cast(CensusMirrors, response.json())
+
+
+def _resolve_census_locator(locator: CensusLocator, mirror: CensusMirror) -> CensusLocator:
```

Comment:
This function resolves the `relative_uri` field against the mirror, and returns a new `CensusLocator` object with the resolved absolute uri. If this feels hacky, we can introduce a new type.